ksql-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.java

```diff
@@ -304,18 +304,19 @@ public void testPrintQueries() {
           + "  \"statementText\" : \"e\",\n"
           + "  \"queries\" : [ {\n"
           + "    \"queryString\" : \"select * from t1\",\n"
-          + "    \"sinks\" : [ \"Test\" ],\n"
+          + "    \"sinkNames\" : [ \"Test\" ],\n"
```

Comment:
check with the C3 people whether or not we need to keep this backwards compatible (i.e. keep the name `sinks` instead of `sinkNmaes`)

Reply:
good point, I'll keep the original to avoid inadvertently breaking the frontend